src/analyze/analyze.c

```diff
@@ -1039,7 +1039,7 @@ static int analyze_critical_chain(int argc, char *argv[], void *userdata) {
         }
         unit_times_hashmap = h;
 
-        (void) pager_open(arg_pager_flags);
```

Comment:
i'd probably change pager_open() to actually return void. And then change the one place where we actually check the return value in cgls.c to use page_have() for what it's doing.